admin_manual/configuration_server/occ_command.rst

```diff
@@ -1434,6 +1434,10 @@ authentication servers such as LDAP::
  | user directories | 2  |
  +------------------+----+
```

Comment:
Could you also update the example output? It does not contain active&disabled lines which seems to always be there.